src/couch/src/couch_httpd_auth.erl

```diff
@@ -186,6 +188,26 @@ proxy_auth_user(Req) ->
             end
     end.
 
+jwt_authentication_handler(Req) ->
+    Secret = config:get("couch_httpd_auth", "jwt_secret", ""),
+    UserClaim = config:get("couch_httpd_auth", "jwt_user_claim", "sub"),
```

Comment:
the JWT `sub` field is defined as indicating the subject of the token, I don't think it makes sense to make this configurable.